docker/docker_image.go

```diff
@@ -103,3 +107,50 @@ func GetRepositoryTags(ctx context.Context, sys *types.SystemContext, ref types.
 	}
 	return tags, nil
 }
+
+// GetDigest returns the image's digest. (This is a
+// backward-compatible shim method which calls the module-level
+// GetDigest)
+func (i *Image) GetDigest(ctx context.Context) (*digest.Digest, error) {
+	return GetDigest(ctx, i.src.c.sys, i.src.logicalRef)
+}
+
+// GetDigest returns the image's digest
```

Comment:
Maybe something like
```suggestion
// GetDigest returns the image's digest
// NOTE: This optimizes to avoid Docker Hub API limits, and mirror configuration may be ignored (but may be implemented in the future)
```
or implement the mirror lookup the same way `newImageSource` does. (As the only current user, it’s up to you.)